dependencies/pom.xml

```diff
@@ -1385,6 +1386,11 @@
                 <artifactId>mssql-jdbc</artifactId>
                 <version>${version.lib.mssql-jdbc}</version>
             </dependency>
+            <dependency>
+                <groupId>io.vertx</groupId>
+                <artifactId>vertx-core</artifactId>
```

Comment:
Like @tomas-langer said, this must go in root project pom.xml. We don't want to manage this dependency for our customers. It's an internal test dependency.

Reply:
my bad, moved to the correct location